modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryServiceTest.java

```diff
@@ -423,6 +495,50 @@ public void testDeleteObjectEntryHierarchy() throws Exception {
 			objectEntry -> Assert.assertNotNull(
 				_objectEntryService.deleteObjectEntry(
 					objectEntry.getObjectEntryId())));
+
+		// User can Delete bound entries if they can update root
```

Comment:
What do you think about this comment?
//User can delete an object entry _of_ a descendant object definition with the update permission.